### PR TITLE
content-range end should be 1 less than file size

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

- Fixing chunked upload

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- #5392 
- https://github.com/appwrite/sdk-generator/pull/648

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

**Review by PineappleIOnic — APPROVED** [review]

**Review by stnguyen90 — APPROVED** [review]

File: tests/e2e/Services/Storage/StorageBase.php [hunk 1]
```diff
@@ -145,7 +145,7 @@ public function testCreateBucketFile(): array
         ];
         $id = '';
         $curlFile = new \CURLFile('data://' . $mimeType . ';base64,' . base64_encode(@fread($handle, $chunkSize)), $mimeType, 'large-file.mp4');
-        $headers['content-range'] = 'bytes ' . ($counter * $chunkSize) . '-' . min(((($counter * $chunkSize) + $chunkSize) - 1), $size) . '/' . $size;
+        $headers['content-range'] = 'bytes ' . ($counter * $chunkSize) . '-' . min(((($counter * $chunkSize) + $chunkSize) - 1), $size - 1) . '/' . $size;
```
Comment (christyjacob4) [inline tests/e2e/Services/Storage/StorageBase.php:148]: doesn't this need to be updated in our SDKs as well ? 
https://github.com/appwrite/sdk-for-php/blob/master/src/Appwrite/Services/Storage.php#L391 
and all the other SDKs too

**Review by christyjacob4 — APPROVED** [review]
Looks good please create the PR for sdks too

**Comment by lohanidamodar** [comment]
@christyjacob4 there's already a related PR in SDK generator.

**Comment by lohanidamodar** [comment]
@christyjacob4 @stnguyen90 I updated so that current SDKs don't break, and we can remove the support in next breaking version

**Review by christyjacob4 — APPROVED** [review]

**Review by eldadfux — APPROVED** [review]

**Pull request merged**